src/ansys/hps/data_transfer/client/exceptions.py

```diff
@@ -19,7 +19,7 @@
 # LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 # SOFTWARE.
-"""This module provides the base class for all client and server HPS-related errors."""
+"""Provides the base class for all client and server HPS-related errors."""
```

Comment:
In the Summary for the exceptions, four of them have the same description: "Provides client-side related errors." Is this what you want?

Reply:
Addressed in the latest commit. Thanks